go_router/lib/src/go_router_delegate.dart

```diff
@@ -220,25 +221,10 @@ class GoRouterDelegate extends RouterDelegate<Uri>
   @override
   Future<void> setNewRoutePath(Uri configuration) async {
     final config = configuration.toString();
-    _log('going to $config');
+    log.info('going to $config');
     _go(config);
   }
 
-  void _log(Object o) {
-    if (debugLogDiagnostics) debugPrint('GoRouter: $o');
-  }
-
-  // note that we're dumping this even if the debugLogDiagnostics flag is clear;
-  // exceptions should be loud and proud!
-  void _logError(Object err, StackTrace stack) =>
-      FlutterError.dumpErrorToConsole(
```

Comment:
I really love using FlutterError.dumpErrorToConsole. Can we continue to use that *and* the logging package?

Reply:
Done. Although before this change, the stack trace was highlighted in red and printed the entire stack trace (100s of lines)

<img width="1188" alt="Screen Shot 2021-12-15 at 11 51 46 AM" src="https://user-images.githubusercontent.com/1145719/146257101-cae43bc2-900e-467a-ae79-b134c5170a3f.png">

After https://github.com/csells/go_router/pull/225/commits/d1ca19835536f5338cba15ee80bf1a4c7b83a5b6, the output is shortened, but loses the color formatting:

<img width="1191" alt="Screen Shot 2021-12-15 at 11 52 35 AM" src="https://user-images.githubusercontent.com/1145719/146256470-1cb90499-de69-4227-af2d-18db51c6581e.png">

I'm not sure which is better 🤷

Reply:
The current implementation shows the abbreviated stack in red for me:

![image](https://user-images.githubusercontent.com/2568253/146301756-abebce30-5f04-4e42-970d-c54259d3ce39.png)

I'd like to keep that behavior.